_pages/clearance-process/supporting-statement.md

```diff
@@ -15,15 +18,15 @@ A few of the criteria covered in the Supporting Statement include:
 *	Practical utility 
 *	Unnecessary duplication 
 
-## Agency purpose
+### Agency purpose
```

Comment:
We go from an `h1` (the page title) to an `h3` here - it's best to try and not skip heading levels, any reason these can't remain `h2`s?

Reply:
Doing an extra check of H-levels loading in the final content!